### PR TITLE
added darkmatter routing

### PR DESCRIPTION
**Pull request by markdav-is**
tried adding routing for the dark matter namspace so I can see the prod image Ideployed using https://darkmatter.prod.apps.mvpstudio.org

### CODE REVIEW

File: running/ambassador-edge/routing/dark-matter-web/prod.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: darkmatter-prod-mapping
+  namespace: ambassador-edge
+spec:
+  host: darkmatter.prod.apps.mvpstudio.org
+  prefix: /
+  service: darkmatter-web.darkmatter-prod
```
Comment (oliverdain) [inline running/ambassador-edge/routing/dark-matter-web/prod.yml]: you don't have a service by this name.

File: running/ambassador-edge/routing/dark-matter-web/prod.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+# Redirect www to non-www
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: darkmatter-www-dns-mapping
+  namespace: ambassador-edge
+spec:
+  host: www.darkmatter.consulting
+  prefix: /
+  service: darkmatter.consulting
```
Comment (oliverdain) [inline running/ambassador-edge/routing/dark-matter-web/prod.yml]: The service name should always be of the form `<service>.<namespace>`. We don't have a `consulting` namespace and I think you want to create just one service for you server so this should also be `darkmatter-web.darkmatter-prod` once you create the `darkmatter-web` service.

**Review by oliverdain — CHANGES_REQUESTED** [review]
I don't think this is quite right. See the comments in Slack: https://eugenetech.slack.com/archives/GF4G2UYMB/p1592532258047600

File: running/ambassador-edge/routing/dark-matter-web/prod.yml (hunk 1 above)
Comment (markdav-is) [reply running/ambassador-edge/routing/dark-matter-web/prod.yml]: I created a service (hopefully) by applying my service definition

File: running/ambassador-edge/routing/dark-matter-web/prod.yml (hunk 2 above)
Comment (markdav-is) [reply running/ambassador-edge/routing/dark-matter-web/prod.yml]: updated to dark-matter-web.dark-matter-web-prod  Having service.namespace is confusing to me, but I'll adapt.  I'm used to namespace.bla.

**Comment by markdav-is** [comment]
I pushed a few updates, please take a look!

**Review by oliverdain — APPROVED** [review]
nice.

**Pull request merged**